human-biomolecular-data/data-sources.md

```diff
@@ -19,93 +25,134 @@ training:
+When looking for solutions to data transfer, you can check [this](https://rdmkit.elixir-europe.org/data_transfer) documentation.
 
 ### Existing approaches
 
-Mention existing appreaches related to the topic.
+- You can check a list of existing controlled access repositories:
+  - [European Genome-phenome Archive](https://ega-archive.org/)
+  - [Estonian BioBank](https://genomics.ut.ee/en/content/estonian-biobank)
+  - [Dutch Covid-19 Data Portal](https://covid19initiatives.health-ri.nl/p/Dashboard) 
+  - [PANTHER](https://pantherstudy.org.uk/) 
+  - [ACE](https://co-connect.ac.uk/ace-cohort/)
+- You can use one of these standards to make your data use conditions publicly available to possible data requesters.
+  - The [GA4GH Data Use Ontology DUO](https://github.com/EBISPOT/DUO) is an international standard, which provides codes to represent data use restrictions for controlled access datasets.
+  - The [Automatable Discovery and Access Matrix (ADA-M)](https://github.com/ga4gh/ADA-M) provides a standardised way to unambiguously represent the conditions related to data discovery and access. 
+  - By depositing your data to one of the existing controlled access repositories, they will already show the data use conditions (e.g. [EGAD00001007777](https://ega-archive.org/datasets/EGAD00001007777))
+- A data access committee (DAC) is a group responsible for reviewing and approving requests for access to sensitive data, such as human biomolecular data. Its role is to ensure that requests are in compliance with relevant laws and regulations, that data is being used for legitimate scientific purposes, and that privacy and security are being maintained. 
 
-## Existing data sources
+You can find further information about sharing human data [here](https://rdmkit.elixir-europe.org/human_data#sharing-and-reusing-of-human-data).
 
-Using bullet point list style as much as possible, describe when, why and for what purpose a specific data source should be used.
+## Data harmonisation
 
-Please avoid replicating 'generic' guidelines, i.e. those not specific to infectious diseases, here.
+To ensure that researchers can effectively utilise data, it is essential that it be collected and stored in a standardised manner. Using standardised formats and databases facilitates data sharing across research groups, enabling more efficient and effective analysis. This approach not only saves time, but also yields more accurate results.
 
-Avoid making long lists of links to data sources.
+It is important to note that there are currently no specific standards and schemes for biomolecular data focused on infectious diseases. Nevertheless, thanks to the Sars-CoV-2 outbreak, the scientific community has established standards, schemes, and data models with controlled vocabularies and ontologies explained in detail in the Human Clinical and Health Data section.
```

Comment:
I think there is, and to our luck, pretty recent: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9950378/

Reply:
The link has been added